package.json

```diff
@@ -27,6 +27,7 @@
     "csvtojson": "~2.0.10",
     "json-stable-stringify": "^1.0.1",
     "kuzzle-plugin-commons": "https://github.com/kuzzleio/kuzzle-plugin-commons/archive/refs/tags/1.0.4.tar.gz",
+    "lodash.isequal": "^4.5.0",
```

Comment:
You can directly import the entirety of lodash since it's already included in Kuzzle dependencies it won't change anything.